web/src/styles/sections/nav-bar.scss

```diff
@@ -28,14 +28,31 @@ $icon-offset: 45px;
   z-index: 5;
 }
 
+.Mui-focusVisible.nav-menu-item-container {
+  background-color: $transparent;
+}
+
+.Mui-focusVisible.nav-menu-item-container.bg-primary-extra-light-mui-selected-override {
+  background-color: $primary-extra-light;
+}
+
+.Mui-focusVisible.profile-non-selected {
+  background-color: #fbfdf9;
+}
+
```

Comment:
Same thing here. All these classes can be nested.
```
.Mui-focusVisible{
  .nav-menu-item-container {
    background-color: $transparent;
    bg-primary-extra-light-mui-selected-override {
      background-color: $primary-extra-light;
    }
  }
 ...
}
```

Reply:
Ya, will fix, forgot we were using scss